docs/GoogleSubscriber-streamingsource.md

```diff
@@ -47,6 +47,7 @@ Naming Convention for Subscription:
        The special characters in the above list can be used in resource names without URL-encoding. 
        However, you must ensure that any other special characters are properly encoded/decoded when used in URLs. 
        For example, mi-tópico is an invalid subscription-name. However, mi-t%C3%B3pico is valid.
+       https://cloud.google.com/pubsub/docs/admin
```

Comment:
Link to the Resource names section of the Pub/Sub docs: https://cloud.google.com/pubsub/docs/admin#resource_names
Also, this should be at the top, maybe linked to Naming Convention, something like: [Naming Convention](https://cloud.google.com/pubsub/docs/admin#resource_names).

Reply:
added more direct doc location suggested and added link to "Naming Convention fore Subscription:" section